README.md

```diff
@@ -2,19 +2,134 @@
 
 ## Bring 'Code as Data' to market with enterprises
 
+### Enable engine integration with 3rd parties
+
+- [ ] gitbase-spark-connector SQL 3rd party interop (Spark Thrift Server) [P0][DP]
+
+### Increase number of algorithms we can run
+
+- [ ] Support for diff tree [P0][DP]
+- [ ] Support for diff [P0][DP]
+- [ ] Gitbase support for low-level git UDFs [P1][DP]
+- [ ] Research options for gitbase plugins [P1][DP]
+- [ ] Research options for UDFs (gitbase or gitbase-spark-connector) [P1][DP]
+
+### Increase variety of data we can analyze
+
+- [ ] Provide specs for external data sources [P1][DP][PRODUCT]
+- [ ] Provide specs to expose dependency analysis [P2][LA]
+
+### Analyze current performance limits and improve them
+
+- [ ] Parallelized queries inside of a repository [P1][DP]
+- [ ] Parallelized index creation [P1][DP]
+- [ ] Make performance dashboard available to product team [P1][DP][PRODUCT]
+- [ ] Provide specs for distributed version (gitbase-spark-connector) [P2][DP]
+
+### Make enterprise friendly
+
+- [ ] gitbase-web SSO and RBAC [P1][APPS][PRODUCT]
+- [ ] bblfsh-web SSO and RBAC [P1][APPS][PRODUCT]
+- [ ] document gitbase audit logs and RBAC [P1][DP]
+
+### Simplify usage
+
+- [ ] Gitbase to expose only reachable objects [P0][DP]
+- [ ] Provide installers [P2][APPS][PRODUCT]
```

Comment:
installers of what? How is it different from engine that is an "installer" for most of src-d projects already?

Reply:
I had the same doubt. Needs clarification.

Reply:
Needs product discussion. This was initially meant to be apt-get and homebrew, but we still need to see if this would actually make things easier or not.